test/test_links.py

```diff
@@ -1,12 +1,12 @@
 from pathlib import Path
 
-from fpdf import FPDF, HTMLMixin
+from fpdf import FPDF
 from test.conftest import assert_pdf_equal
 
 HERE = Path(__file__).resolve().parent
 
 
-class PDF(FPDF, HTMLMixin):
+class PDF(FPDF):
```

Comment:
This class is useless then, please just replace `PDF` usages by `FPDF`

Reply:
It's fixed at 11d7d248b5ddf08d9e9af5510674d85e1ba0aea3